### ISSUE TITLE
429 responses can occur during indexing - bulk index queue gets full

### ISSUE THREAD

**Issue by roborourke** [humanmade/altis-enhanced-search#231]
This is probably more of an issue for lower envioronments using `t3.small` Elasticsearch instances but will cause indexing to stop.

We could experiment with adding a sleep timeout to the below action and make it configurable using an environment variable and/or filter:

```
do_action( 'ep_cli_' . $indexable->slug . '_bulk_index', $objects, $response );
```

More info here https://aws.amazon.com/premiumsupport/knowledge-center/resolve-429-error-es/

This might be improved by upgrading elasticsearch to 6.3 or higher.

**Comment by joehoyle** [comment]
I don't think we need a timeout, I think we just need to detect the 429, then sleep, then retry the request (rather than slowing everything down).

**Comment by joehoyle** [comment]
> This might be improved by upgrading elasticsearch to 6.3 or higher.

I'm seeing this on 7.10 too
